### DISCUSSION TITLE
More than one ip connections for my system, one asking installer password and one not asking

### DISCUSSION THREAD

**Discussion by mad-jack** [wlcrs/huawei_solar#799] [Connecting to the inverter]
Hi everyone 
I have a SUN2000-5KTL-L1 with firmware V200R001C00SPC130, with Luna2000 10kW battery. I'm successful connect all things with HA huawei_solar by wifi dongle and all it seems working ok.
But looking on my router DHCP I've found 2 different IP, on the same network, for the solar system. At first only one was working on port 502 and slave ID 1 . It works with (or without) elevate permission without asking any installator password.
The, playing with installator APP and activating  "local O&M" setting on wifi connection, I discovered that the second IP is working on huawei_solar. In that case it works on port 6607 on ID 0 and it ask installator credentials for elevate permission. No difference on any other things, entities or  device were the same.
Have anyone  notice that or any other differences?

**Comment by Roving-Ronin** [comment]
Are you saying your sDongle is one connection and then you've also got the inbuilt WiFi of the L1 also connecting to the home WiFi ?
Disable the inbuilt WiFi and just enable to sDongle for mobus access.

**Comment by mad-jack** [comment]
No, I say that on the sDongle connection the router finds 2 interfaces. Then the router DHCP assign two different IP addresses. One IP works with integration without password, while the other works only if the "localO&M"  is on , and works on port 6607 with id 0 and asking installator username and password .

**Comment by Roving-Ronin** [comment]
WiFi is disabled and Ethernet connected? Somehow is WiFi and Ethernet connected at the same time (should be either only, not both as an option) as that should be the only way the sDongle could appear with two IPs.... assuming they have different MAC addresses for each?

**Comment by mad-jack** [comment]
No ethernet, only wifi, but on modbus-TCP dongle wifi configuration, I set restricted connection only to hassie IP. You are right on mac-address the inverter/dongle show 2 mac address. One starting with 10:a4:xx:xx:xx:xx and the other starting with 3c:54:xx:xx:xx:xx . The router recognize every address coming from Huawei manufacturer , the first with the inverter name + serial, the second unknown Huawei device. 
May be that the modbus shows a direct (I mean unrestricted by password) interface if you restrict the quering IP to only one ip.
